packages/app/pages/AgentRelationshipRole/AgentRelationshipRole.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import allAgentRelationshipRoles from "../../../ui-bindings/AgentRelationshipRole/AllAgentRelationshipRoles";
+import agentRelationshipRoleById from "../../../ui-bindings/AgentRelationshipRole/AgentRelationshipRoleById";
```

Comment:
I prefaced my files with "get" we should be consistent. do you want to add get? or do you think it is better form to not include that?

Reply:
Good naming convention! I changed my file names to match the viewer queries they were exporting
:+1: